README.md

```
@@ -4,163 +4,235 @@ A set of functions for the **F**riendly **I**nteractive **Sh**ell (fish). This i
- - **`🥚`**: alias for `eggs`;
- - **`🤖`**: alias for `groq`;
- - **`🎥`**: alias for `movie`;
- - **`🍅`**: alias for `pomodoro`;
- - **`💼`**: alias for `pomodoro`;
- - **`😴`**: alias for `sleep_for`;
- - **`⌚`**: alias for `smartwatch`;
- - **`🦷`**: alias for `teeth`;
- - **`🌄`**: alias for `wakeup`;
- - **`🚶`**: alias for `walking`; and
- - **`👕`**: alias for `washingmachine`.
+  - **`🚨`**: alias for `alarm`;
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)